rio_rgbify/mbtiler.py

```diff
@@ -88,12 +90,12 @@ def _encode_as_png(data, profile, dst_transform):
     contents: bytearray
         png-encoded bytearray of the provided input data
     """
-    profile['affine'] = dst_transform
+    profile["affine"] = dst_transform
 
-    with rasterio.open('/vsimem/tileimg', 'w', **profile) as dst:
+    with rasterio.open("/vsimem/tileimg", "w", **profile) as dst:
         dst.write(data)
 
-    contents = bytearray(virtual_file_to_buffer('/vsimem/tileimg'))
+    contents = bytearray(virtual_file_to_buffer("/vsimem/tileimg"))
```

Comment:
This function should use rasterio's MemoryFile instead.

Reply:
as mentioned in the previous comment, I didn't want to touch to any of the code in this PR, I'm happy to make the code more 2019 alike in another PR